src/reader.rs

```diff
@@ -3,12 +3,11 @@ use std::{env, fs, path::PathBuf};
 
 /// Read seeds from specified file
 pub fn read_file(filename: &str, base_dir: &str) -> Result<String> {
-    let mut path = PathBuf::new();
-    if let Ok(current_base) = env::var("CARGO_MANIFEST_DIR") {
-        path.push(current_base)
-    }
-    path.push(base_dir);
-    path.push(filename);
+    let path = env::var("CARGO_MANIFEST_DIR")
+        .map(PathBuf::from)
+        .unwrap_or_default()
+        .join(base_dir)
+        .join(filename);
```

Comment:
that's really neat ❤️

Reply:
I understand.
Looking forward to having your pr merged!